lib/components/Actions/Actions.tsx

```diff
@@ -12,24 +12,19 @@ interface Props
 	extends Partial<
 		Pick<ComponentProps<typeof Columns>, 'noWrap' | 'wrappingDirection'>
 	> {
-	className?: string;
```

Comment:
I checked mfe, no actions has ever had classNames sent to it. And in the spirit of removing this as we progress. I've opted to remove that in this case.